src/cve_sources/nvd_cve.py

```diff
@@ -61,17 +60,30 @@ def fetch_cves(invch):
                 (elem for elem in description_data if elem["lang"] == "en"),
                 description_data[0],
             )["value"]
-            
+
 
             # First matching keyword or False if no keyword matches (generator empty)
             keyword = next(
-                (
-                    key
-                    for key in invch.inventory
-                    if key["keyword"].lower() in description.lower()
-                ),
-                False,
+                (key for key in invch.inventory if re.search(rf'\b{re.escape(key["keyword"].lower())}\b', description, re.IGNORECASE)),
+                False
+            )
+ 
+            matched_docker_compose = next(
```

Comment:
Why are docker-compose images processed in a dedicated step? Don't they provide the same container_last_seen metric and labels as the Kubernetes ones?